blues/ncmc.py

```diff
@@ -173,15 +173,14 @@ def __init__(self, temperature, residueList, **kwds):
         self.dummy_simulation = None
         self.nc_context = None
         self.nc_integrator = None
-        self.nc_pos = None
         self._storage = None
         self.temperature = temperature
         kB = unit.BOLTZMANN_CONSTANT_kB * unit.AVOGADRO_CONSTANT_NA
         kT = kB * temperature
         beta = 1.0 / kT
         self.beta = beta
 
-    def get_particle_masses(self, system, residueList=None):
+    def get_particle_masses(self, system, set_self=True, residueList=None):
```

Comment:
(e.g., what does it do if this is/is not true? I should be able to use it without having to read the code...)